stdlib/bytes.gr

```diff
@@ -385,6 +385,7 @@ provide let getInt8U = (index: Number, bytes: Bytes) => {
   let offset = coerceNumberToWasmI32(index)
   checkIndexIsInBounds(offset, _INT8_BYTE_SIZE, size)
   let n = WasmI32.load8U(ptr + offset, _VALUE_OFFSET)
+  // TODO change
```

Comment:
Should change.

Reply:
Completely forgot about this, my bad 😅 

Also unrelated to this PR: since unsigned 32/64 bit ints exist now should `getInt32, getInt64` be removed in favor of `getInt32U, getInt32S, ...`?

Reply:
Conveniently the type is called `Int32` so those can stay the same 🙂 
There could be some new ones called `getUint32`, etc.

Reply:
Should I tack `getUint32`, etc into this PR or make a separate one?

Reply:
A separate PR would be better.

Reply:
Ok should I update the 8/16-bit portions of Buffer/Bytes in this PR or make all of the changes in a separate PR after this one is merged for the sake of consistency?

Reply:
Yeah, might as well do it in a new PR.